integrations/examples/imports-and-exports.md

```diff
@@ -16,7 +16,7 @@ First we are going to want to initialize a new project. To do this we can naviga
 {% tabs %}
 {% tab title="Rust" %}
 {% hint style="info" %}
-The final code for this example can be found on [GitHub](https://github.com/wasmerio/wasmer/blob/master/examples/instance.rs).
+The final code for this example can be found on [GitHub](https://github.com/wasmerio/wasmer/blob/master/examples/improts_exports.rs).
```

Comment:
`imports_exports` rather than `improts` :-).

Reply:
Fixed in https://github.com/wasmerio/docs.wasmer.io/commit/dad8868e635bfa83fc735685dfb841ccf5d22078.